diff --git a/hack/build-image/Dockerfile b/hack/build-image/Dockerfile
index b959299..9917e9c 100644
--- a/hack/build-image/Dockerfile
+++ b/hack/build-image/Dockerfile
@@ -69,7 +69,8 @@ RUN ARCH=$(go env GOARCH) && \
chmod a+x /usr/include/google/protobuf && \
chmod a+r -R /usr/include/google && \
chmod +x /usr/bin/protoc
-RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.32.0
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.33.0 \
+ && go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3.0
# get goreleaser
# goreleaser name template per arch is basically goarch except for amd64 and
386
https://github.com/goreleaser/goreleaser/blob/ec8819a95c5527fae65e5cb41673f5bbc3
245fda/.goreleaser.yaml#L167C1-L173C42
diff --git a/hack/update-2proto.sh b/hack/update-2proto.sh
index 8ad696e..b2a9c18 100755
--- a/hack/update-2proto.sh
+++ b/hack/update-2proto.sh
@@ -19,6 +19,15 @@ HACK_DIR=$(dirname "${BASH_SOURCE}")
echo "Updating plugin proto"
echo protoc --version
-protoc pkg/plugin/proto/*.proto pkg/plugin/proto/*/*/*.proto --
go_out=plugins=grpc:pkg/plugin/generated/ --go_opt=module=github.com/vmware-
tanzu/velero/pkg/plugin/generated -I pkg/plugin/proto/ -I /usr/include
-
+#protoc pkg/plugin/proto/*.proto pkg/plugin/proto/*/*/*.proto --
go_out=plugins=grpc:pkg/plugin/generated/ --go_opt=module=github.com/vmware-
tanzu/velero/pkg/plugin/generated -I pkg/plugin/proto/ -I /usr/include
+protoc \
+ -I pkg/plugin/proto/ \
+ -I /usr/include \
+ --go_out=pkg/plugin/generated/ \
+ --go_opt=module=github.com/vmware-tanzu/velero/pkg/plugin/generated \
+ --go-grpc_out=pkg/plugin/generated \
+ --go-grpc_opt=paths=source_relative \
+ --go-grpc_opt=require_unimplemented_servers=false \
+ $(find pkg/plugin/proto -name '*.proto')
+
echo "Updating plugin proto - done!"